src/components/Checkbox/index.js

```diff
@@ -1,56 +1,40 @@
 /** @jsx jsx */
-import React from 'react';
 import { jsx } from '@emotion/core';
+import { useCallback, useState } from 'react';
+
 import Text from 'components/Text';
 import { TEXT_TYPE } from 'components/Text/constants';
 import { ReactComponent as Check } from 'static/icons/check.svg';
 
 import styles from './Checkbox.styles';
 
-class InputCheckbox extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      checked: false,
-    };
-  }
+function InputCheckbox({ label, customOnChange, ...rest }) {
```

Comment:
Are there pros and cons for using named functions vs `const`? I have no opinion on the matter, I'm just curious.

Reply:
Named functions appear on error stacks better than arrow functions, so if a component any issue, the error messages are better to digest.